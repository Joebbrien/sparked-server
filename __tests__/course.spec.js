import axios from "axios";
import dotenv from "dotenv";

dotenv.config();

describe("courses resolvers", () => {
  test("should not query courses when not authenticated", async () => {
    const responseData = await axios.post(process.env.URL, {
      query: `
        query {
          getCourses {
            name
            _id
          }
        }
      `
    });
    const {
      data: { data, errors }
    } = responseData;

    expect(data.getCourses).toBe(null);
    expect(errors[0].message).toBe("you must be logged in");
  });
});
