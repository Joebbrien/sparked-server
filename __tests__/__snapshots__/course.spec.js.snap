// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`courses resolvers should create a course and return proper data 1`] = `
Object {
  "createdAt": "1559203783917",
  "createdBy": "5cef8859d21cb0007e2893bd",
  "name": "Another Introduction",
}
`;
